light_curves/multiband_lc.ipynb

```diff
@@ -95,7 +97,7 @@
     "    get_green_sample, get_lyu_sample, get_lopeznavas_sample, get_hon_sample, get_yang_sample,  get_SDSS_sample, clean_sample\n",
     "from ztf_functions import ZTF_get_lightcurve\n",
     "from data_structures import MultiIndexDFObject\n",
-    "from heasarc_functions import HEASARC_get_lightcurves\n",
+    "from heasarc_functions import HEASARC_get_lightcurves, make_VOTable\n",
```

Comment:
`make_VOTable` import is not needed in the notebook

Reply:
removed it.